operator-common/src/test/java/io/strimzi/operator/common/OperatorMetricsTest.java

```diff
@@ -300,6 +303,61 @@ protected Status createStatus() {
                 })));
     }
 
+
+    @Test
+    @SuppressWarnings({ "unchecked", "rawtypes" })
+    public void testFailingReconcileWithClientExceptionNotBlocked(VertxTestContext context)  {
```

Comment:
I don't think this really tests metrics. Can we find some better class for this? Or does it depend on some shared code in this class?